src/modules/integrator-guide/pages/automation/workflows/examples/template-rendering.adoc

```diff
@@ -0,0 +1,164 @@
+**** *value*: `renderedTemplate.document`
+** *results*:
+*** *document target*: `content`
+====
+****
+
+[CAUTION]
+====
+Make sure that the `documentType` parameter of the `(2) Render template` is of value `text/html`
+====
+
+== PDF
+The difference between HTML and PDF rendering is the `documentType` parameter changes to `application/pdf`. With PDFs you can also adjust the xref:integrator-guide:templates/index.adoc#tpl-render-options_marginBottom[render options] to modify how the final PDF is rendered.
+
+[IMPORTANT]
+====
+In order to render templates as PDF documents, xref:devops-guide:extension-requirements/pdf-renderer.adoc[PDF rendering must be setup].
+====
+
+.The screenshot outlines the workflow that renders a PDF template and sends it as an email attachment.
+image::workflow/example-template-rendering/pdf-workflow.png[role="data-zoomable"]
+
+The link:{attachment-path}template_rendering_pdf.json[source code] for the workflow example.
+
+****
+.Workflow step details:
+[%collapsible.result]
+====
+1. *(1) System; onManual*:
+** *resource*: `System`
+** *event*: `onManual`
+** *enabled*: checked
+2. *(2) Render template*:
+** *type*: `Template render`
+** *arguments*:
+*** *lookup*:
+**** *type*: `Handle`
+**** *value type*: constant
+**** *value*: `pdf-template`
+*** *documentName*:
+**** *value type*: constant
+**** *value*: `PDF template`
+*** *documentType*:
+**** *value type*: constant
+**** *value*: `application/pdf`
+*** *options*:
+**** *type*: `renderOptions`
+**** *value type*: expression
+**** *value*: `{
+  "documentSize": "A4",
+  "contentScale": "1",
+  "orientation": "portrait",
+  "margin": "0.3"
+}`
+** *results*:
+*** *document target*: `renderedTemplate`
+3. *(3) Build email*:
+** *type*: `Email builder`
+** *arguments*:
+*** *subject*:
+**** *type*: `String`
+**** *value type*: constant
+**** *value*: `PDF template`
+*** *from*:
+**** *type*: `String`
+**** *value type*: constant
+**** *value*: `test@mail.com`
+*** *to*:
+**** *type*: `String`
+**** *value type*: constant
+**** *value*: `example@mail.com`
+*** *html*:
+**** *type*: `Reader`
+**** *value type*: expression
+**** *value*: `content.document`
+** *results*:
+*** *message target*: `email`
+4. *(4) Attach PDF template*:
+** *type*: `Attach PDF template`
+** *arguments*:
+*** *message*:
+**** *type*: `EmailMessage`
+**** *value type*: expression
+**** *value*: `email`
+*** *content*:
+**** *type*: `Reader`
+**** *value type*: expression
+**** *value*: `renderedTemplate.document`
+*** *name*:
+**** *type*: `String`
+**** *value type*: constant
+**** *value*: `PDF template.pdf`
+5. *(5) Send email*:
+** *type*: `Email sender`
+** *arguments*:
+*** *message*:
+**** *type*: `EmailMessage`
+**** *value type*: expression
+**** *value*: `email`
+====
+****
+
+[TIP]
+====
+You can combine both types of templates to render a dynamic email that has a dynamic PDF attached to it. Example xref:integrator-guide:automation/workflows/examples/email-attachment.adoc[here]
```

Comment:
One sentence per line.

End with period (`.`)

Can you try to rewrite this bit as I find it a bit "awkward" to read; something like this (you can also copy-paste my version if you see it fit)

```
You can combine both types of templates to render dynamic emails with PDF attachments.
The source code for the workflow can be found here
```